tools-local/tasks/FinalizeBuild.cs

```diff
@@ -97,6 +97,8 @@ public override bool Execute()
                     CopyBlobs($"{Channel}/Installers/{SharedFrameworkNugetVersion}", $"{Channel}/Installers/Latest/");
 
                     CopyBlobs($"{Channel}/Installers/{SharedHostNugetVersion}", $"{Channel}/Installers/Latest/");
+                    
+                    CopyBlobs($"{Channel}/Checksums/{SharedFrameworkNugetVersion}", $"{Channel}/Checksums/Latest/");
```

Comment:
Actually, checksums should not be necessary in 'Finalize' at all.  This should be removed.

---
In reply to: [116110189](https://github.com/dotnet/core-setup/pull/2368#discussion_r116110189) [](ancestors = 116110189)